### PR TITLE
Remove full stops from query explanation clauses

### PR DESCRIPTION
**Pull request by chrisdavenport**
#### Summary of Changes

Some language strings in Smart Search are intended to be used as clauses within a constructed sentence.  Although they look like sentences in their own right, this is not how they are used and hence they should not end in full stops.

The sentence is assembled here: https://github.com/joomla/joomla-cms/blob/staging/components/com_finder/helpers/html/query.php

These language strings are not used anywhere else.
#### Testing Instructions
- Make sure you have Smart Search enabled and some content indexed.
- Make sure you have "Query explanation" set to "Yes" in a menu item or in the global options.
- Enter a search term (eg. "the").  The query explanation given will be something like this: "Assuming **the** is required., the following results were found.".   Note the full stop after "required".
- Apply this PR.
- Repeat the search and you should see: "Assuming **the** is required, the following results were found.".
- Similarly for the other clauses.

### CODE REVIEW

File: language/en-GB/en-GB.com_finder.ini [hunk 1]
```diff
@@ -28,11 +28,11 @@ COM_FINDER_QUERY_OPERATOR_NOT="not"
...
-COM_FINDER_QUERY_TOKEN_EXCLUDED="<span class="_QQ_"term"_QQ_">%s</span> should be excluded."
+COM_FINDER_QUERY_TOKEN_EXCLUDED="<span class="_QQ_"term"_QQ_">%s</span> should be excluded"
 COM_FINDER_QUERY_TOKEN_GLUE=", and "
 COM_FINDER_QUERY_TOKEN_INTERPRETED="Assuming %s, the following results were found."
-COM_FINDER_QUERY_TOKEN_OPTIONAL="<span class="_QQ_"term"_QQ_">%s</span> is optional."
-COM_FINDER_QUERY_TOKEN_REQUIRED="<span class="_QQ_"term"_QQ_">%s</span> is required."
+COM_FINDER_QUERY_TOKEN_OPTIONAL="<span class="_QQ_"term"_QQ_">%s</span> is optional"
+COM_FINDER_QUERY_TOKEN_REQUIRED="<span class="_QQ_"term"_QQ_">%s</span> is required"
 COM_FINDER_SEARCH_NO_RESULTS_BODY="No search results could be found for query: %s."
 COM_FINDER_SEARCH_NO_RESULTS_BODY_MULTILANG="No search results (in English-UK) could be found for query: %s."
```
Comment (Hils) [inline language/en-GB/en-GB.com_finder.ini]: English-UK : we have en-GB not English-UK (I think)

Comment (chrisdavenport) [inline language/en-GB/en-GB.com_finder.ini]: That may be, but it's not the subject of this PR.  Please submit a new PR for that.  Thanks.

**Comment by Hils** [comment]
Will do! I just get confused sometimes when things are committed with no tests. Sorry!

**Comment by brianteeman** [comment]
I have tested this item :white_check_mark: successfully on 3dbf27cd2107f85cd377629f83c6b729883dce0c<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/9957">issues.joomla.org/joomla-cms/9957</a>.</sub>

**Comment by brianteeman** [comment]
Setting RTC as it is an obvious fix<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/9957">issues.joomla.org/joomla-cms/9957</a>.</sub>

**Pull request merged**